tests/test_kvdb.py

```diff
@@ -31,7 +31,8 @@ def test_sync(kvdb: hse.Kvdb):
 
 
 def test_param(kvdb: hse.Kvdb):
-    assert kvdb.param("throttling.init_policy") == '"default"'
+    assert (kvdb.param("throttling.init_policy") == '"heavy"' or
```

Comment:
What are your thoughts on presenting the configured `auto` back to the user?

Reply:
Not even sure how feasible that is.

Reply:
Makes sense to me.

Reply:
Agreed and yes it returns what's in effect...